Reference documentation for the Window Controls Overlay API (#11368)
mdn/content: files/en-us/web/api/windowcontrolsoverlay/gettitlebararearect/index.md

---
title: WindowControlsOverlay.prototype.getTitleBarAreaRect()
slug: Web/API/WindowControlsOverlay/getTitleBarAreaRect
tags:
  - API
  - Method
  - Reference
  - WindowControlsOverlay
  - Progressive Web Apps
browser-compat: api.WindowControlsOverlay.getTitleBarAreaRect
---
{{APIRef("Window Controls Overlay API")}}{{SeeCompatTable}}

> The **`WindowControlsOverlay.prototype.getTitleBarAreaRect()`** method queries the current geometry of the title bar area of the desktop Progressive Web App window.

Reviewer: ```suggestion
The **`getTitleBarAreaRect()`** method of the {{domxref('WindowControlsOverlay')}} interface queries the current geometry of the title bar area of a desktop Progressive Web App window.
```

## Syntax

```js

> navigator.windowControlsOverlay.getTitleBarAreaRect();

Reviewer: ```suggestion
getTitleBarAreaRect();
```

Reviewer: Per the [current instructions](https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods).

---

> title: WindowControlsOverlay.prototype.getTitleBarAreaRect()

Reviewer: ```suggestion
title: WindowControlsOverlay.getTitleBarAreaRect()
```

Reviewer: I apologize for net catching this on the first review. We don't do this any more because 'prototype' wont' appear in a script. If there's a reason to distinguish between static and instance methods on a particular interface it's added to the basic description. For example, "The doStuff() static method of the whatever interface...".
